packages/projection/package.json

```diff
@@ -49,7 +49,6 @@
     "tsc-alias": "^1.8.10"
   },
   "devDependencies": {
-    "@cardano-sdk/ogmios": "workspace:~",
```

Comment:
Looks like my vscode sorted `package.json` :facepalm:

Reply:
This is also fixed (but I forget about this change as is mix with all the other changes in the fixup 😬 )